docs/destinations/forwarding-events/google-tag-manager-server-side/index.md

```diff
@@ -28,6 +28,12 @@ Use GTM SS to relay enriched events to destinations. Events are sent to GTM SS a
 * For Community Edition, see [Snowbridge](/docs/destinations/forwarding-events/snowbridge/index.md).
 * For Snowplow BDP, you can [request setup](https://console.snowplowanalytics.com/destinations/catalog) through the Console.
 
+:::note
+
+Destinations Hub is the recommended way to setup GTM Server-side because it allows to take full advantage of the Snowplow pipeline and forward validated and enriched data to downstream destinations.
```

Comment:
```suggestion
Destinations Hub is the recommended way to setup GTM Server-side because it allows you to take full advantage of the Snowplow pipeline, and forward validated and enriched data to downstream destinations.
```